pkg/sysinfo/sysinfo.go

```diff
@@ -50,17 +47,6 @@ func New(quiet bool) *SysInfo {
 		}
 	}
 
-	// Checek if ipv4_forward is disabled.
-	if data, err := ioutil.ReadFile("/proc/sys/net/ipv4/ip_forward"); os.IsNotExist(err) {
```

Comment:
Since https://github.com/docker/docker/pull/3801 enabled ip_forward by default, this warning will only shown when we disabled ip-forward, which as we always consider unnecessary. (eg. when we disable swap memory, we don't want warning about swap memory is disabled.)
If you mean some extreme situation like `/proc/sys/net/ipv4/ip_forward` somehow doesn't exist, we already have the same warning when we trying to write `1` to that file.